### PR TITLE
Add an idiom

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by jserv** [comment]
The git commit message and content are inconsistent with each other.

File: data/tsi.src [hunk 1]
```diff
@@ -92343,6 +92343,7 @@
 狗尾續貂 0 ㄍㄡˇ ㄨㄟˇ ㄒㄩˋ ㄉㄧㄠ
 狗屁 26 ㄍㄡˇ ㄆㄧˋ
 狗屁不通 5 ㄍㄡˇ ㄆㄧˋ ㄅㄨˋ ㄊㄨㄥ
+狗屁倒灶 10 ㄍㄡˇ ㄆㄧˋ ㄉㄠˇ ㄗㄠˋ
```
Comment (PeterDaveHello) [inline data/tsi.src]: I thought this is what I've done and record in the commit message, did I miss or misunderstand anything here?

**Comment by PeterDaveHello** [comment]
@jserv I'm not sure how should I change the commit message, would you please give me your advice about that? Thanks!

File: data/tsi.src (hunk 1 above)
Comment (jserv) [inline data/tsi.src]: The subject uses `ㄆㄧˊ` while the body takes "ㄆㄧˋ", that is inconsistent. Please use `git commit --amend` to rework.

Comment (PeterDaveHello) [inline data/tsi.src]: My bad ... sorry

**Comment by PeterDaveHello** [comment]
Updated, sorry for my careless behavior.

**Pull request merged**